offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -411,4 +408,13 @@ class CreatePersonView extends VerticalLayout {
         createPersonViewModel.outdatedPerson = null
 
     }
+
```

Comment:
It would be great if you could provide javadoc for public and protected methods since they are part of the API of your class.

Reply:
+1